utils/wasm-builder/src/wasm_project.rs

```diff
@@ -338,6 +348,16 @@ fn project_enabled_features(
 	enabled_features
 }
 
+/// Returns if the project has the `runtime-wasm` feature
+fn has_runtime_wasm_feature(
```

Comment:
Nit: A naming that includes "declares" (e.g. `has_runtime_wasm_feature_declared`) might be a bit better, since "having feature" is a bit vague in general.

Reply:
Maybe this would have helped my slow brain.